doc/nrf/ug_nrf5340.rst

```diff
@@ -106,165 +85,246 @@ Zephyr includes the `OpenAMP`_ library, which provides a complete solution for e
+
+* :option:`CONFIG_BT_RPMSG_NRF53` - set to ``y`` in all Bluetooth LE samples
+* :option:`CONFIG_NRF_802154_SER_HOST` - set to ``y`` in all Thread, Zigbee, and Matter samples
 
-+-----+--------------------------+-------------------------------+--------------------------------------------+
-|     |``CONFIG_BT_RPMSG_NRF53`` |``CONFIG_NRF_802154_SER_HOST`` | Child image sample for the network core    |
-+=====+==========================+===============================+============================================+
-|`a)` |``y``                     |``n``                          |:ref:`zephyr:bluetooth-hci-rpmsg-sample`    |
-+-----+--------------------------+-------------------------------+--------------------------------------------+
-|`b)` |``n``                     |``y``                          |:ref:`zephyr:nrf-ieee802154-rpmsg-sample`   |
-+-----+--------------------------+-------------------------------+--------------------------------------------+
-|`c)` |``y``                     |``y``                          |:ref:`multiprotocol-rpmsg-sample`           |
-+-----+--------------------------+-------------------------------+--------------------------------------------+
-|`d)` |``n``                     |``n``                          |some other possibilities, not defined here  |
-+-----+--------------------------+-------------------------------+--------------------------------------------+
+The combination of these options determines which (if any) sample is included in the build for the application sample as a child image for the ``nrf5340_dk_nrf5340_cpunet`` core:
 
-`a)` This option is used for all Bluetooth Low Energy samples in the |NCS| since in this case :option:`CONFIG_BT_RPMSG_NRF53` defaults to ``y``.
++---------------------------------+--------------------------------------+--------------------------------------------+
+| :option:`CONFIG_BT_RPMSG_NRF53` | :option:`CONFIG_NRF_802154_SER_HOST` | Child image sample for the network core    |
++=================================+======================================+============================================+
+|``y``                            |``n``                                 |:ref:`zephyr:bluetooth-hci-rpmsg-sample`    |
++---------------------------------+--------------------------------------+--------------------------------------------+
+|``n``                            |``y``                                 |:ref:`zephyr:nrf-ieee802154-rpmsg-sample`   |
++---------------------------------+--------------------------------------+--------------------------------------------+
+|``y``                            |``y``                                 |:ref:`multiprotocol-rpmsg-sample`           |
++---------------------------------+--------------------------------------+--------------------------------------------+
+|``n``                            |``n``                                 |some other possibilities, not defined here  |
```

Comment:
Should we add the entry for when `empty_app_core` is added by the network core sample? Maybe in another table?

Reply:
What options does this depend on? Only CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE?

Reply:
Yes, that option has the requried dependencies built into it, so IMO just mentioning that option should be sufficient.

Reply:
Thanks! Updated.